### PR TITLE
undoing dirty fix

### PR DESCRIPTION
**Pull request by FBartos**
fixes: https://github.com/jasp-stats/jasp-test-release/issues/2292

### CODE REVIEW

**Review by vandenman — APPROVED** [review]

**Comment by boutinb** [comment]
> @boutinb I'm not 100% sure what the best way forward is but Cochrane cannot be tested without this PR. Should we merge this and immediately create new nightlies, or should we wait with the new nightlies?

Let's merge this. The buildbot should make automatically a new build on Windows. I can quickly make a build on Mac ARM. Over 1 hour I will go for holidays...

**Comment by juliuspfadt** [comment]
Nice! so the testing responsibility then lies with someone else? Cause I am on macOS intel...

File: R/cochranecommon.R [hunk 1]
```diff
@@ -19,6 +19,7 @@
 # Most of functions implemented here provide data search functionality.
 #' @import jaspBase
 #' @importFrom jaspDescriptives .plotMarginal
+#' @importFrom jaspMetaAnalysis .ClassicalMetaAnalysisCommon .BayesianMetaAnalysisCommon
```
[the remark this answers is not in the document]
Comment (FBartos) [reply R/cochranecommon.R:22]: hmm, it actually generated the namespace for me

**Comment by boutinb** [comment]
So just done also a Mac Intel version.... now I leave. See you over 2 weeks.

**Pull request merged**